### ISSUE TITLE
Learn more -> source, reliability, and completeness

### ISSUE THREAD

**Issue by edgarcosta** [LMFDB/lmfdb#5675]
At the moment, we point to https://www.lmfdb.org/knowledge/show/rcs

should we instead point to a page that includes the knowl?

That is what we do for:
`www.lmfdb.org/Database/{Source,Completeness,Reliability}`

**Comment by AndrewVSutherland** [comment]
Yes I think we should.  But I don't know what you mean by referring to www.lmfdb.org/Database/{Source,Completeness,Reliability} as an example, none of those links work for me.

**Comment by edgarcosta** [comment]
`Database` is for example
 `ModularForm/GL2/Q/holomorphic/` or `EllipticCurve/Q/`

**Comment by roed314** [comment]
I still don't understand what this issue is about.  "we point to https://www.lmfdb.org/knowledge/show/rcs" from where?

**Comment by roed314** [comment]
Where are we displaying the knowl page? I think my confusion is that I
don't know what page this issue is about.

**Comment by edgarcosta** [comment]
Let me try again.
Assume that you are a new user, and you see the box "Lean more" on the homepage
>  Information is available regarding the [source, reliability, and completeness](https://www.lmfdb.org/knowledge/show/rcs) of the database.

that link will take you directly to a knowl page, where you also see things like:
- Authors, Knowl status, Referred to by, History, etc...

while if you go to, https://www.lmfdb.org/EllipticCurve/Q/ and click on any of the Learn more links

[Source and acknowledgments](https://www.lmfdb.org/EllipticCurve/Q/Source)
    [Completeness of the data](https://www.lmfdb.org/EllipticCurve/Q/Completeness)
    [Reliability of the data](https://www.lmfdb.org/EllipticCurve/Q/Reliability)
    [Elliptic curve labels](https://www.lmfdb.org/EllipticCurve/Q/Labels)

you get into a webpage that embeds the contents of the knowl, without explicitly exposing that to the user, and thus less confusing.

**Comment by roed314** [comment]
Ok, so you just want the front page link to be wrapped in `single.html`.  Sounds reasonable to me.
